frontend/src/Login.css

```
@@ -362,6 +364,13 @@
     color: black;
   }
 
+  .login-title {
+    -webkit-text-fill-color: currentColor;
+    -webkit-background-clip: initial;
+    background-clip: initial;
+    background: none;
```

Stylelint (17.12.0)

[error] Overridden property "background-clip" by shorthand "background" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)